root/static/scripts/common/components/TagEditor.js

```diff
@@ -203,11 +203,11 @@ class TagEditor extends React.Component<TagEditorProps, TagEditorState> {
 
   debouncePendingVotes: () => void;
 
-  genreMap: {+[string]: GenreT, ...};
+  genreMap: {+[genre: string]: GenreT, ...};
 
   genreNames: $ReadOnlyArray<string>;
 
-  pendingVotes: {[string]: PendingVoteT, ...};
+  pendingVotes: {[tagName: string]: PendingVoteT, ...};
```

Comment:
`tag.name` is used at https://github.com/metabrainz/musicbrainz-server/blob/974e52b1fb7619b6fdcf2eb73de53771a071b9fe/root/static/scripts/common/components/TagEditor.js#L401